deploy/resources/optional/tasksnippets/buildah-pr-task-snippet.yaml

```diff
@@ -27,7 +27,7 @@ spec:
             resource: app-image
         params:
           - name: BUILDER_IMAGE
-            value: quay.io/buildah/stable:v1.11.0
+            value: registry.redhat.io/rhel8/buildah
```

Comment:
Any reason for not using a tag ?

Reply:
We aren't using tag in cluster-task also, that's why I didn't add tag here. Should we add tag here as well as in clustertask?
@nikhil-thomas @piyush-garg